source/development/packaging/generate-ova.rst

```diff
@@ -32,33 +32,29 @@ Execute the ``generate_ova.sh`` script, with the different options you desire.
 .. code-block:: none
   :class: output
 
-  Usage: ./generate_ova.sh [OPTIONS]
-
-    -v, --version          [Required] Version of wazuh to install on VM.
-    -o, --opendistro       [Required] Version of Open Distro for Elasticsearch.
-    -f, --filebeat         [Required] Filebeat's version.
-    -r, --repository       [Required] Select the software repository [prod/dev].
-    -b, --branch           [Optional] Branch/tag of the Wazuh repository.
-    -s, --store <path>     [Optional] Set the destination absolute path of package.
-    -c, --checksum <path>  [Optional] Generate checksum.
-    -u, --ui-revision      [Optional] Revision of the UI package. By default, 1.
-    -h, --help             [  Util  ] Show this help.
+  General usage: generate_ova.sh [OPTIONS]
+    -w,    --wazuh            [Optional] Select the wazuh major version [4.1, 4.2]. By default: 4.2
+    -r,    --repository       [Optional] Select the software repository [prod/dev]. By default: prod
+    -s,    --store <path>     [Optional] Set the destination absolute path where the ova file will be stored.
+    -c,    --checksum         [Optional] Generate checksum [yes/no]. By default: no
+    -g,    --debug            [Optional] Set debug mode on [yes/no]. By default: no
+    -h,    --help             [  Util  ] Show this help.
 
 The options for the repository indicates whether the packages used to install Wazuh are the production ones or not.
 
- * Stable: The OVA uses released packages.
- * Unstable: The OVA uses not released packages.
+ * prod: The OVA uses released packages.
+ * dev: The OVA uses not released packages.
```

Comment:
```suggestion
 * dev: The OVA uses pre-release packages.
```

Reply:
Fixed in https://github.com/wazuh/wazuh-documentation/pull/4150/commits/9df2633702e37b1a3c1c1642d89251a4097ec848